view/mail-body.html

```diff
@@ -1,749 +1,119 @@
+
+<!--[if (gte mso 9)|(IE)]>
+<table cellspacing="0" cellpadding="0" border="0" width="600" align="center" role="presentation"><tr><td>
+<![endif]-->
+<div role="article" lang="en" style="background-color: white; color: #2b2b2b; font-family: -apple-system, BlinkMacSystemFont, Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI', 'Segoe UI Emoji', 'Segoe UI Symbol'; font-size: 18px; font-weight: 400; line-height: 28px; margin: 0 auto; max-width: 600px; padding: 40px 20px 40px 20px; overflow-wrap: break-word; word-wrap: break-word">
+
+  <header>
+    <!-- banner -->
+      <a href="https://github.com/ory/{{.RepoName}}" target="_blank"><img alt="ORY {{.RepoName}}" src="https://raw.githubusercontent.com/ory/meta/master/static/banners/{{.RepoName}}.png" width="600" border="0" style="display: block; max-width: 100%; min-width: 100px; width: 100%;"></a>
+    <!-- version header -->
+      <h1 style="color: #000000; font-size: 24px; font-weight: 500; line-height: 32px; margin: 16px 0; text-align: center;">
+        <a href="https://github.com/ory/{{.RepoName}}/releases/tag/{{.GitTag}}" target="_blank">{{.ProjectName}} {{.Version}}</a><br>
+        <span style="font-size: 16px; font-weight: 400;">is here!</span><br>
+        <span style="font-size: 16px; font-weight: 400;"><a href="https://www.ory.sh/{{.RepoName}}" target="_blank">Website</a>&nbsp;| <a href="https://github.com/ory/{{.RepoName}}" target="_blank">GitHub</a> |&nbsp;<a href="https://github.com/ory/{{.RepoName}}/blob/master/CHANGELOG.md" target="_blank">Changelog</a>&nbsp;|&nbsp;<a href="https://github.com/ory/{{.RepoName}}/blob/master/UPGRADE.md" target="_blank">Upgrade Guide</a><br></span>
+      </h1>
+  </header>
+
+
+  <main>
+    <!-- auto generated content-->
+    <article>
+      <br>
+      {{if .Message }}{{ .Message }}<br>
+      {{ end -}}<br>
+      <br>
+      {{.Changelog -}}<br>
+      <br>
+      Changes requiring your attention may have been introduced since the last release, please read the&nbsp;<a href="https://github.com/ory/{{.RepoName}}/blob/master/UPGRADE.md" target="_blank">upgrade guide</a>&nbsp;with care.
```

Comment:
```suggestion
      Changes requiring your attention may have been introduced since the last release, please read the&nbsp;<a href="https://github.com/ory/{{.RepoName}}/blob/master/CHANGELOG.md" target="_blank">changelog</a>&nbsp;with care.
```